spec/type_aliases/cron/minute_entry_spec.rb

```diff
@@ -5,7 +5,9 @@
     it { is_expected.to allow_value('22')}
     it { is_expected.to allow_value('*')}
     it { is_expected.to allow_value('*/5')}
+    it { is_expected.to allow_value('2/5')}
```

Comment:
why is this now allowed?  crontab allows '<range>/<skip> and '2' is not a range

Reply:
> Two other minor nits:
> 
> * REFERENCE.md needs to be regenerated once the code is finalized
> * Windows acceptance test is failing in GitLab

The windows acceptance tests just timed out connecting like what happens with the ssh on linux.  I restarted it and it complete correctly

Reply:
From the man page on systemd.timer  2/5 is valid.  It will just skip until valid values are exhausted.  2..10/5 will stop at the upper value of the range.